src/panels/PatientControlPanel.jsx

```diff
@@ -110,25 +128,31 @@ class PatientControlPanel extends Component {
                             <Col xs={3} lg={2} className='logo-title-column'>
                                 {this.renderFluxNotesLogo()}
                             </Col>
-                            <Col xs={5} md={4} lg={4} className="summary-header-column">
+                            <Col xs={4} md={4} lg={3} className="summary-header-column">
                                 {this.renderSummaryHeader()}
                             </Col>
-
+                            <Col xs={1} lg={1}>
+                                {this.renderSelectPatientArrow()}
+                            </Col>
                             <Col xs={4} md={5} lg={6}>
                                 {this.renderConditionSelectAndSearch()}
                             </Col>
                         </Row>
                     </Grid>
                 </Paper>
+                {this.isTablet && <PatientSelectionModal isModalOpen={this.state.isModalOpen} isTablet={this.isTablet} dataAccess={this.props.dataAccess} handleClose={this.handleClose} loadPatient={this.props.loadPatient}/>}
             </div>
+
         );
     }
 }
 
 PatientControlPanel.propTypes = {
     appTitle: PropTypes.string.isRequired,
```

Comment:
[JA3]: I think you also added loadPatient as a prop to PatientControlPanel, so I think you should reflect that in the propTypes.

Reply:
Looks good! Are any of these required props that you can add `.isRequired` onto?

Reply:
yes, thanks!  Resolved!

Reply:
Oh no! I think I led you astray here and `dataAccess` and `loadPatient` are actually not required because they don't get passed into this component on the other endpoints, like on /pilot1. Now we get prop warnings when you load that endpoint.

Reply:
oops, fixed!